### PR TITLE
Adds DISA SCAP content

### PR DESCRIPTION
**Pull request by dalerichardson**
Includes updates from the following DISA SCAP Benchmarks:

* DotNet_Framework_4-0_V2R2_STIG_SCAP_1-2_Benchmark
* MS_IE11_V2R5_STIG_SCAP_1-2_Benchmark
* MS_Windows_10_V2R8_STIG_SCAP_1-2_Benchmark
* MS_Windows_2012_and_2012_R2_DC_V3R5_STIG_SCAP_1-2_Benchmark
* MS_Windows_2012_and_2012_R2_MS_V3R5_STIG_SCAP_1-2_Benchmark
* MS_Windows_Server_2016_V2R4_STIG_SCAP_1-2_Benchmark
* MS_Windows_Server_2019_V2R4_STIG_SCAP_1-2_Benchmark
* RHEL_7_V3R11_STIG_SCAP_1-2_Benchmark
* RHEL_8_V1R9_STIG_SCAP_1-2_Benchmark

### CODE REVIEW

File: scap/content/guides/disa/stig-el7-scap_1-2.xml [hunk 1]
```diff
@@ -7388,6 +7388,8 @@ Remove any duplicate or conflicting lines from /etc/sudoers and /etc/sudoers.d/
             <criterion comment="RHEL 7 Workstation is installed" test_ref="oval:mil.disa.stig.rhel7:tst:18" />
             <criterion comment="RHEL 7 Server is installed" test_ref="oval:mil.disa.stig.rhel7:tst:19" />
             <criterion comment="RHEL 7 Compute Node is installed" test_ref="oval:mil.disa.stig.rhel7:tst:20" />
+            <criterion comment="CentOS 7 is installed" test_ref="oval:mil.fso.centos.centos7:tst:220" />
```
Comment (lorengordon) [inline scap/content/guides/disa/stig-el7-scap_1-2.xml]: This line can be deleted.

```suggestion
```

File: scap/content/guides/disa/stig-el7-scap_1-2.xml [hunk 2]
```diff
@@ -12303,6 +12313,12 @@ This requirement addresses concurrent sessions for information system accounts a
         <rpminfo_object xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#linux" id="oval:mil.disa.stig.rhel7:obj:58" version="1">
           <name>redhat-release-computenode</name>
         </rpminfo_object>
+        <rpminfo_object xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#linux" id="oval:mil.fso.centos.centos7:obj:258" version="1">
+          <name>centos-release</name>
+        </rpminfo_object>
```
Comment (lorengordon) [inline scap/content/guides/disa/stig-el7-scap_1-2.xml]: This block can be deleted

```suggestion
```

File: scap/content/guides/disa/stig-el7-scap_1-2.xml [hunk 3]
```diff
@@ -15205,6 +15243,12 @@ This requirement addresses concurrent sessions for information system accounts a
         <rpminfo_state xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#linux" id="oval:mil.disa.stig.rhel7:ste:59" version="1">
           <version operation="pattern match">^7.*$</version>
         </rpminfo_state>
+        <rpminfo_state xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#linux" id="oval:mil.fso.centos.centos7:ste:259" version="1">
+          <version operation="pattern match">^7.*$</version>
+        </rpminfo_state>        
+        <rpminfo_state xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#linux" id="oval:mil.fso.centos.centos7:ste:259" version="1">
+          <version operation="pattern match">^7.*$</version>
+        </rpminfo_state>
```
Comment (lorengordon) [inline scap/content/guides/disa/stig-el7-scap_1-2.xml]: This block can be deleted

```suggestion
```

**Review by lorengordon — CHANGES_REQUESTED** [review]

File: scap/content/guides/disa/stig-el7-scap_1-2.xml (hunk 3 above)
Comment (dalerichardson) [reply scap/content/guides/disa/stig-el7-scap_1-2.xml]: Removed

File: scap/content/guides/disa/stig-el7-scap_1-2.xml (hunk 1 above)
Comment (dalerichardson) [reply scap/content/guides/disa/stig-el7-scap_1-2.xml]: Removed

File: scap/content/guides/disa/stig-el7-scap_1-2.xml [hunk 4]
```diff
@@ -10852,6 +10854,14 @@ This requirement addresses concurrent sessions for information system accounts a
           <object object_ref="oval:mil.disa.stig.rhel7:obj:58" />
           <state state_ref="oval:mil.disa.stig.rhel7:ste:59" />
         </rpminfo_test>
+        <rpminfo_test xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#linux" check="all" check_existence="at_least_one_exists" comment="centos-release is version 7" id="oval:mil.fso.centos.centos7:tst:220" version="1">
+          <object object_ref="oval:mil.fso.centos.centos7:obj:258" />
+          <state state_ref="oval:mil.fso.centos.centos7:ste:259" />
+        </rpminfo_test>
```
[the remark this answers is not in the document]
Comment (dalerichardson) [reply scap/content/guides/disa/stig-el7-scap_1-2.xml]: Removed

File: scap/content/guides/disa/stig-el7-scap_1-2.xml [hunk 5]
```diff
@@ -7387,7 +7387,8 @@ Remove any duplicate or conflicting lines from /etc/sudoers and /etc/sudoers.d/
             <criterion comment="RHEL 7 Client is installed" test_ref="oval:mil.disa.stig.rhel7:tst:17" />
             <criterion comment="RHEL 7 Workstation is installed" test_ref="oval:mil.disa.stig.rhel7:tst:18" />
             <criterion comment="RHEL 7 Server is installed" test_ref="oval:mil.disa.stig.rhel7:tst:19" />
-            <criterion comment="RHEL 7 Compute Node is installed" test_ref="oval:mil.disa.stig.rhel7:tst:20" />
+            <criterion comment="RHEL 7 Compute Node is installed" test_ref="oval:mil.disa.stig.rhel7:tst:20" />           
+            <criterion comment="CentOS Enterprise Linux 7 is installed" test_ref="oval:mil.fso.centos.centos7:tst:220" />
```
Comment (lorengordon) [inline scap/content/guides/disa/stig-el7-scap_1-2.xml]:
```suggestion
```

Comment (lorengordon) [inline scap/content/guides/disa/stig-el7-scap_1-2.xml]:
```suggestion
```

File: scap/content/guides/disa/stig-el7-scap_1-2.xml [hunk 6]
```diff
@@ -10852,6 +10853,10 @@ This requirement addresses concurrent sessions for information system accounts a
           <object object_ref="oval:mil.disa.stig.rhel7:obj:58" />
           <state state_ref="oval:mil.disa.stig.rhel7:ste:59" />
         </rpminfo_test>
+        <rpminfo_test xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#linux" check="all" check_existence="at_least_one_exists" comment="centos-release is version 7" id="oval:mil.fso.centos.centos7:tst:220" version="1">
+          <object object_ref="oval:mil.fso.centos.centos7:obj:258" />
+          <state state_ref="oval:mil.fso.centos.centos7:ste:259" />
+        </rpminfo_test>
```
Comment (lorengordon) [inline scap/content/guides/disa/stig-el7-scap_1-2.xml]: This block should be:

```suggestion
        <rpminfo_test xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#linux" check="all" check_existence="at_least_one_exists" comment="redhat-release-computenode is version 7" id="oval:mil.fso.centos.centos7:tst:220" version="1">
          <object object_ref="oval:mil.fso.centos.centos7:obj:258" />
          <state state_ref="oval:mil.fso.centos.centos7:ste:259" />
        </rpminfo_test>
```

**Review by lorengordon — APPROVED** [review]

**Pull request merged**